tasks/copy_to_archive_adapter/copy_to_archive_adapter.py

```diff
@@ -0,0 +1,86 @@
+"""
+Name: copy_to_archive.py
+Description: Lambda function that takes a Cumulus message, extracts a list of files,
+and copies those files from their current storage location into a staging/archive location.
+"""
+import json
+import os
+from typing import Any, Dict, List, Union
+
+# Third party libraries
+import boto3
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+# noinspection PyPackageRequirements
+from botocore.client import Config
+from run_cumulus_task import run_cumulus_task
+
+OS_ENVIRON_COPY_TO_ARCHIVE_ARN_KEY = "COPY_TO_ARCHIVE_ARN"
+
+ORCA_INPUT_KEY = "input"
+ORCA_CONFIG_KEY = "config"
+
+# Set AWS powertools logger
+LOGGER = Logger()
+
+
+# noinspection PyUnusedLocal
+def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str, Any]:
+    """
+    Converts event to a format accepted by ORCA's copy_to_archive lambda,
+    then calls copy_to_archive and returns the result.
+
+    Args:
+        event: Passed through from {handler}
+        context: An object required by AWS Lambda. Unused.
+
+    Environment Variables:
+        COPY_TO_ARCHIVE_ARN (string, required):
+            ARN of ORCA's copy_to_archive lambda.
+
+    Returns:
+        A dict representing input and copied files. See schemas/output.json for more information.
+    """
+    # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html
+    # 600s for copy_to_archive operation
+    config = Config(read_timeout=600, retries={'total_max_attempts': 1})
+    client = boto3.client("lambda", config=config)
```

Comment:
This setup will likely need some updates for cross-account access, though it looks like boto3 does support some methods of cross-account. https://bugs.earthdata.nasa.gov/browse/ORCA-633